pkg/server/server_test.go

```diff
@@ -120,15 +118,15 @@ func TestEncapsulated(t *testing.T) {
 // when it's running in bootstrap mode.
 // The test does the following:
 //
-// 1. Fetch the MachineConfigPool from the testdata.
-// 2. Fetch the MachineConfig from the testdata.
-// 3. Manually update the ignition config from Step 2 by adding
-//    the NodeAnnotations file, the kubeconfig file (which is read
-//    from the testdata). This Ignition config is then
-//    labeled as expected Ignition config.
-// 4. Call the Bootstrap GetConfig method by passing the reference to the
-//    MachineConfigPool present in the testdata folder.
-// 5. Compare the Ignition configs from Step 3 and Step 4.
+//  1. Fetch the MachineConfigPool from the testdata.
```

Comment:
This comment change is due to gofmt changes in recent go versions, I can undo it if that's preferred